docs/source/_templates/index_PKG.html

```diff
@@ -21,7 +21,7 @@
 `GeoIPS License <https://github.com/NRLMMD-GEOIPS/geoips/blob/main/LICENSE>`__ |
 `NRLMMD <https://www.nrlmry.navy.mil>`__ |
```

Comment:
Hmm, is there any way we can access the documentation built by the CI, so we can actually look at the results of these changes without building it locally?  That would be really nice ...

Reply:
Yes, there is! I'll message you on Slack.